src/components/sidebar/Sidebar.vue

```diff
@@ -122,7 +127,7 @@
 
             <!-- Add a warning if this phyloreference has changed -->
             <ModifiedIcon
-              message="This phyloreference has been modified since being loaded! Use 'Save as JSON' to save your changes."
+              message="This phyloreference has been modified since being loaded! Use 'Save' to save your changes."
```

Comment:
Why _being loaded_ instead of _was loaded_ (as used above)? Though indeed, _since it was loaded_ is poor grammar, it should either be _since it has been loaded_ or _after it was loaded_. Can you fix this throughout?

Reply:
Fixed in 92a9c8d!